File: live_stream_app/lib/modules/stream/model/stream_request.dart
class StreamRequest {
  StreamRequest({
      this.userId, 
      this.firstName, 
      this.lastName, 
      this.birthDay, 
      this.address, 
      this.email, 
      this.file, 
      this.title, 
      this.content,});

  StreamRequest.fromJson(dynamic json) {
    userId = json['userId'];
    firstName = json['firstName'];
    lastName = json['lastName'];
    birthDay = json['birthDay'];
    address = json['address'];
    email = json['email'];
    file = json['file'];
    title = json['title'];
    content = json['content'];
  }
  int? userId;
  String? firstName;
  String? lastName;
  String? birthDay;
  String? address;
  String? email;
  String? file;
  String? title;
  String? content;

  Map<String, dynamic> toJson() {
    final map = <String, dynamic>{};
    map['userId'] = userId;
    map['firstName'] = firstName;
    map['lastName'] = lastName;
    map['birthDay'] = birthDay;
    map['address'] = address;
    map['email'] = email;
    map['file'] = file;
    map['title'] = title;
    map['content'] = content;
    return map;
  }

}

File: spring-boot-refresh-token-jwt-master/src/main/java/com/bezkoder/spring/security/jwt/payload/request/StreamRequest.java
package com.bezkoder.spring.security.jwt.payload.request;

import javax.validation.constraints.Email;
import javax.validation.constraints.NotBlank;
import javax.validation.constraints.NotNull;
import javax.validation.constraints.Size;

/**
 * Created by Benjamin Lam on 3/30/2022.
 * SELF EDU
 * binhlam10796@gmail.com
 */
public class StreamRequest {

    @NotNull
    private Long userId;

    @NotBlank
    @Size(min = 3, max = 222)
    private String firstName;

    @NotBlank
    @Size(min = 3, max = 222)
    private String lastName;

    @NotBlank
    @Size(min = 3, max = 255)
    private String birthDay;

    @NotBlank
    @Size(min = 3, max = 255)
    private String address;

    @NotBlank
    @Size(max = 50)
    @Email
    private String email;

    @NotBlank
    @Size(min = 3, max = 255)
    private String file;

    @NotBlank
    @Size(min = 3, max = 255)
    private String title;

    @NotBlank
    @Size(min = 3)
    private String content;

    public Long getUserId() {
        return userId;
    }

    public void setUserId(Long userId) {
        this.userId = userId;
    }

    public String getTitle() {
        return title;
    }

    public void setTitle(String title) {
        this.title = title;
    }

    public String getContent() {
        return content;
    }

    public void setContent(String content) {
        this.content = content;
    }

    public String getFirstName() {
        return firstName;
    }

    public void setFirstName(String firstName) {
        this.firstName = firstName;
    }

    public String getLastName() {
        return lastName;
    }

    public void setLastName(String lastName) {
        this.lastName = lastName;
    }

    public String getBirthDay() {
        return birthDay;
    }

    public void setBirthDay(String birthDay) {
        this.birthDay = birthDay;
    }

    public String getAddress() {
        return address;
    }

    public void setAddress(String address) {
        this.address = address;
    }

    public String getEmail() {
        return email;
    }

    public void setEmail(String email) {
        this.email = email;
    }

    public String getFile() {
        return file;
    }

    public void setFile(String file) {
        this.file = file;
    }
}
